ignite/cmd/chain_build.go

```diff
@@ -72,8 +74,23 @@ func chainBuildHandler(cmd *cobra.Command, _ []string) error {
 		return err
 	}
 
+	cacheRootDir, err := chainconfig.ConfigDirPath()
```

Comment:
I see that this and following code duplicated among other commands, can we make it reusable?

Here is an idea:

- We implement a new func in `cmd.go`
```go
func newCache(combra.Command) (cache.Cache, error)
``` 

This func will create the cache, and also clear the cache if the flag is set.

Usage:
```go
cache, error := newCache(cmd)
if err != nil {
    return err
}
defer cache.Close() // I think we can instruct this to close immediately after we create cache. otherwise we may forget calling it in the end of the handler.
```

Reply:
Makes sense. I did this and now this is the only way it is instantiated currently.